lib/api/bucketPutVersioning.js

```diff
@@ -75,6 +79,16 @@ function _checkBackendVersioningImplemented(bucket) {
     return true;
 }
 
+function _isValidReplicationVersioning(result, bucket) {
+    const status = result.VersioningConfiguration.Status[0];
```

Comment:
Is it guaranteed here that `VersioningConfiguration` always exist in the result? Even if so I would rather check it to be sure.